src/engraving/compat/pageformat.h

```diff
@@ -0,0 +1,85 @@
+/*
+ * SPDX-License-Identifier: GPL-3.0-only
+ * MuseScore-CLA-applies
+ *
+ * MuseScore
+ * Music Composition & Notation
+ *
+ * Copyright (C) 2021 MuseScore BVBA and others
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+#ifndef MU_ENGRAVING_READPAGEFORMAT_H
+#define MU_ENGRAVING_READPAGEFORMAT_H
+
+#include "draw/geometry.h"
+
+namespace Ms {
+class MStyle;
+class XmlReader;
+}
+
+namespace mu::engraving::compat {
+//---------------------------------------------------------
+//   @@ PageFormat
+//---------------------------------------------------------
+
+class PageFormat
+{
+    mu::SizeF _size;
+    qreal _printableWidth;          // _width - left margin - right margin
```

Comment:
might make sense to initialize values here

Reply:
as it was, so I keep, this is an obsolete class, it is better to change it less and it is not worth wasting time